report/Report.cpp

```diff
@@ -128,6 +128,24 @@ bool is_in(std::string str, std::vector<std::string> &vec) {
   return std::find(vec.begin(), vec.end(), str) != vec.end();
 }
 
+/**
+ * @brief Get the filename from a path
+ * @param path: path to get filename from, like /src/proj/filename.cc
+ * @return filename like filename.cc is path is correct, otherwise path
+ */
+std::string get_filename_from_path(const std::string &path) {
```

Comment:
Is this function used by anyone?

Reply:
was used to set file name, not needed anymore